### PR TITLE
Set background color for map preview loading div to #F5F5F5; closes #…

### LINKED ISSUE #1596: Add background colour to map preview
### Problem description
When the preview map is loading only the spinner is shown, leaving an awkward gap in the page flow. 

### Proposed solution
Adding a subtle background colour to the loading preview map will prepare the eye for that content to appear once it has finished loading, making the sequence seem smoother (hopefully)

Background HEX: #F5F5F5

### Problem reproduction steps

1. Navigate to https://dev.magda.io/dataset/ds-nsw-188ae132-d7b4-4a1a-8785-21786bcbe0bf/details?q=
2. Scroll to see preview map loading

### Screenshot / Design / File reference

screenshot 2018-08-14 16 31 35

### PR DESCRIPTION
**Pull request by sajidanower23**
### What this PR does

Fixes #1596 

Set a background color for loading screens.

### Checklist

-   [x] unit tests aren't applicable
-   [x] I've updated CHANGES.md with what I changed.
-   [x] I've linked this PR to an issue in ZenHub (core dev team only)

FireShot Capture 002 - Interim Biogeographic Regionalisation for Australia (IBRA), Version 7_ - localhost

### CODE REVIEW

File: magda-web-client/src/_variables.scss [hunk 1]
```diff
@@ -9,7 +9,7 @@ $magda-color-dark: darken($magda-color-primary, 15%);
 $AU-color-foreground-action: $magda-color-light;
 $AU-color-foreground-focus: $magda-color-dark;
 $AU-color-foreground-text: $magda-color-dark;
-$AU-color-background: #ffffff;
```
Comment (t83714) [inline magda-web-client/src/_variables.scss]: If we change here, probably gonna impact a lot of places. 
e.g. search box, search result page etc.
image
I think we only want to update the background for the loading area? @philipgrimmett  please comment on this (here is a test site: https://1596-map-preview-background.dev.magda.io/)

**Comment by t83714** [comment]
Also, when you've got a chance, could we please update the `CHANGES.md` as well?
It's only a tiny change. But might be useful in future to know when the colour has been changed~

File: magda-web-client/src/_variables.scss (hunk 1 above)
Comment (philipgrimmett) [reply magda-web-client/src/_variables.scss]: @t83714 yep, my intention was to only apply the grey background to loading components like the embedded map. I'd suggest doing the same for other components you expect might take some time to load (in the absence of actual skeleton screens). Ping @alyce

Comment (sajidanower23) [reply magda-web-client/src/_variables.scss]: @t83714 Yep, that's fair. I have added a new color instead for preview components.

**Comment by t83714** [comment]
@sajidanower23 
Tested locally and works great now~
Good Job~  👍 
Noticed you haven't updated the contributor.md so added your name in~

**Pull request merged**